### DISCUSSION TITLE
Product and Team naming conventions

### DISCUSSION THREAD

**Discussion by rpshaw** [idaholab/moose#19065] [Q&A General] [answered]
I know this is not a technical question, so if there is a better venue, I'm all ears. I work at Sandia Labs on the Sierra suite of codes, which is somewhat similar to MOOSE in scope and functionality. I had some question about naming conventions:

- I was wondering how you came up with the module names: bison, falcon, etc. 
- Do you structure your teams around the modules? 
  - If not, do they have team names and how did those team names come about?

It seems like there is some continuity with your naming that we'd like to explore on our side.

Thanks,
Ryan

**Comment by GiudGiud** [comment]
@friedmud

**Comment by friedmud** [comment, marked as answer]
I don't think it's all that different from Sierra naming - which shouldn't be surprising since I came from Sandia, working on Sierra 😄 

FALCON, BISON, Pronghorn, etc. are applications that utilize the framework and are effectively equivalent to Adagio, Aria, etc. at Sandia.  They are code names - and they are typically the names of the code teams (depends on the size of the application - some are small and are only developed by one person).

When we can we try to make them acronyms and have them be animals from Idaho.  MOOSE came first (Multiphysics Object-Oriented Simulation Environment) then BISON (recursive acronym: BISON Implicit Simulation Of Nuclear-Fuel), etc.  But - many (like Pronghorn) are just animal names and aren't acronyms.

The animal names have proven a good choice since people all over the world can pick animal names that are meaningful to them (like the mascot of their school, or an animal that is only in their area, etc.)

Let me know if somehow I missed the intent of your question!
